### ISSUE TITLE
Postman crashing on startup...

### ISSUE THREAD

**Issue by charliekwong** [a85/POSTMan-Chrome-Extension#3230]
## Issue Report:
 1. clicked to update to latest versio 5.0.2 - restart and it crashes almost  immediately - I can see that initially I am offline but no collections (all disappeared)
then pop-up appears saying "electron has stopped working"

I have uninstalled 5.0.2 and found 4.9.2 setup.exe and installed that - but same problem.
 
any help appreciated.

**Comment by charliekwong** [comment]
Postman starts and stays up long enough that I can see that I have about 10 tabs open. the history and collections are empty - I am offline but my name is in the top corner...I guess it must be trying to sync ?

**Comment by charliekwong** [comment]
Problem solved.

**Comment by sdnts** [comment]
Hi @charliekwong Out of curiosity, what was causing this?

**Comment by charliekwong** [comment]
Hi Sidhant
I don’t know. If you want I can send you the one of the many crashdump files that Postman puts into C:\Users\....\AppData\Local\CrashDumps and you might be able to determine from there.

What I did to solve the problem was to make sure of a full uninstall:
Because it still knew who I was when I uninstalled and re-installed 4.9.2 or 5.0.2, using the windows control panel, I suspected that the uninstall was not clean.
There is no uninstall postman. From the squirelltmp log I can see that it failed to install the uninstaller.
So I  delete the folders
C:\Users\....\AppData\Local\squirreltmp
C:\Users\....\AppData\Local\postman
C:\Users\u698489\AppData\Roaming\Postman

Re-installed 5.0.2 and hey – it is a clean install, and I had to re-login and got my collections back.

Charlie
